libs/akita/src/lib/queryEntity.ts

```diff
@@ -291,22 +291,28 @@ export class QueryEntity<S extends EntityState, EntityType = getEntityType<S>, I
    * Listen for entity actions
    *
    *  @example
-   *
    *  this.query.selectEntityAction(EntityActions.Add);
    *  this.query.selectEntityAction(EntityActions.Update);
    *  this.query.selectEntityAction(EntityActions.Remove);
    *
+   *  this.query.selectEntityAction([EntityActions.Add, EntityActions.Update, EntityActions.Remove])
+   *
    *  this.query.selectEntityAction();
    */
   selectEntityAction(action: EntityActions): Observable<IDType[]>;
+  selectEntityAction(actions: EntityActions[]): Observable<IDType[]>;
   selectEntityAction(): Observable<EntityAction<IDType>>;
-  selectEntityAction(action?: EntityActions): Observable<IDType[] | EntityAction<IDType>> {
-    if (isUndefined(action)) {
+  selectEntityAction(actionOrActions?: EntityActions | EntityActions[]): Observable<IDType[] | { [Name in EntityActionsNames]?: IDType[] } | EntityAction<IDType>> {
+    if (isNil(actionOrActions)) {
       return this.store.selectEntityAction$;
     }
+
+    const project = isArray(actionOrActions) ? ({ type, ids }: EntityAction<IDType>) => ({ [type]: ids }) : ({ ids }: EntityAction<IDType>) => ids;
```

Comment:
@NetanelBasal Instead of rewriting the `EntityAction` we should just pass it through?
```
    const project = isArray(actionOrActions) ? ((entityAction: EntityAction<IDType>) => entityAction) : ({ ids }: EntityAction<IDType>) => ids;
```

Reply:
Yes